src/Image.cpp

```diff
@@ -116,6 +116,21 @@ struct AlignedDeleter {
     void operator()(T* p) const { delete [] ((char**)p)[-1]; }
 };
 
+template <typename T>
+std::shared_ptr<T> allocateAlignedMemory(int n)
+{
+    // This bit is based on the answers here:
```

Comment:
This bit and the comment make me wonder if I could take advantage of aligned memory in, e.g., batoid.  There I just let numpy do all the allocating.  Any idea if numpy defaults to aligning things on 16 byte boundaries?

Reply:
No, numpy doesn't (necessarily) align on 16 byte boundaries.  Maybe it does for complex128, since then the data type itself is 16 byte, but I haven't tested it.  We have a different hack to force 128 byte allocations in the python-layer Image class.  Look at the _make_empty method of Image in python.